data/json/itemgroups/SUS/clothes_store.json

```
@@ -0,0 +1,136 @@
+    "subtype": "distribution",
+    "entries": [
+      { "collection": [ { "group": "shirts_unisex" }, { "group": "pants_unisex" }, { "group": "shoes_unisex" } ], "prob": 95 },
+      {
+        "collection": [ { "group": "shirts_womens" }, { "group": "pants_womens" }, { "group": "shoes_womens" } ],
+        "prob": 5
+      }
+    ]
+  },
+  {
+    "id": "SUS_mannequin_fur",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for a mannequin displaying fur clothes.",
+    "subtype": "collection",
+    "entries": [
+      { "group": "fur_coats_unisex" },
+      { "distribution": [ { "group": "pants_unisex", "prob": 80 }, { "group": "fur_pants_womens", "prob": 20 } ] },
+      { "group": "fur_shoes_unisex" },
+      { "group": "fur_gloves_unisex" },
+      { "group": "fur_hats_unisex" }
+    ]
+  },
+  {
+    "id": "SUS_mannequin_dresses",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for a mannequin displaying dresses.",
+    "subtype": "collection",
+    "entries": [ { "group": "dresses_womens" } ]
+  },
+  {
+    "id": "SUS_mannequin_formal_mens",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for a mannequin displaying formal clothes.",
+    "subtype": "collection",
+    "entries": [ { "group": "formal_set_mens" } ]
+  },
+  {
+    "id": "SUS_tailoring_bookcase",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for a set of tailoring manuals and references.",
+    "subtype": "collection",
+    "entries": [
+      { "item": "mag_tailor", "count": [ 1, 10 ], "prob": 80 },
+      { "item": "manual_tailor", "count": [ 1, 4 ], "prob": 80 },
+      { "item": "tailor_portfolio", "prob": 20 },
+      { "item": "tailor_japanese", "prob": 10 },
+      { "item": "textbook_tailor", "count": [ 1, 4 ], "prob": 80 }
+    ]
+  },
+  {
+    "id": "SUS_tailoring_tool_drawer",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for a set of tailoring tools.",
+    "subtype": "collection",
+    "entries": [
+      {
+        "distribution": [
+          {
+            "collection": [
+              { "item": "awl_steel", "prob": 70 },
+              { "item": "needle_curved", "prob": 75 },
+              { "item": "scissors", "prob": 90 },
+              { "item": "knitting_needles", "prob": 70 },
```

```suggestion
              { "item": "knitting_needles", "prob": 20 },
```
Knitting needles are pretty unlikely in a tailor drawer I think